datadog/resource_datadog_dashboard.go

```diff
@@ -4531,6 +4531,255 @@ func buildTerraformTimeseriesDefinition(datadogDefinition datadogV1.TimeseriesWi
 	return terraformDefinition
 }
 
+func getFormulaSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"formula_expression": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "String expression built from queries, formulas and functions.",
+				},
+				"limit": {
+					Type:        schema.TypeList,
+					Optional:    true,
+					Description: "Options for limiting results returned.",
+					MaxItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"count": {
+								Type:     schema.TypeInt,
+								Optional: true,
+							},
+							"order": {
+								Type:     schema.TypeString,
+								Optional: true,
+								Default:  "desc",
+							},
```

Comment:
Can you add a description to these two

Reply:
Addressed in commit [5c22cd7](https://github.com/DataDog/terraform-provider-datadog/pull/892/commits/5c22cd771533e25ac58a9e87bc9da024672d34f3).